packages/nimble-components/src/tabs/template.ts

```diff
@@ -0,0 +1,75 @@
+import { html, ref, slotted, when } from '@microsoft/fast-element';
+import type { ViewTemplate } from '@microsoft/fast-element';
+import {
+    endSlotTemplate,
+    startSlotTemplate,
+    FoundationElementTemplate,
+    TabsOptions
+} from '@microsoft/fast-foundation';
+import type { Tabs } from '.';
+import { buttonTag } from '../button';
+import { smallPadding } from '../theme-provider/design-tokens';
+import { iconArrowExpanderLeftTag } from '../icons/arrow-expander-left';
+import { iconArrowExpanderRightTag } from '../icons/arrow-expander-right';
+
+// prettier-ignore
+export const template: FoundationElementTemplate<
+ViewTemplate<Tabs>,
+TabsOptions
+> = (context, definition) => html`
+    <template class="${x => x.orientation}">
+        <div
+            ${ref('tabsContainer')}
+            class="tabsrootcontainer"
+            style="
+                --ni-private-tabs-container-padding: ${x => (x.showScrollButtons ? smallPadding.getValueFor(x) : '0px')};
+            "
+        >
+            ${startSlotTemplate(context, definition)}
+            ${when(x => x.showScrollButtons, html<Tabs>`
+                <${buttonTag} 
+                    content-hidden
+                    class="scroll-button left"
+                    appearance="ghost"
+                    tabindex="-1"
+                    @click="${x => x.onScrollLeftClick()}"
+                    ${ref('leftScrollButton')}
+                >
```

Comment:
I suppose there could be some debate on this. The buttons are not reachable via the keyboard, and arguably we could/should opt to simply set `aria-hidden="true"`. Having a tooltip on hover for these buttons doesn't seem that helpful, and accidentally pressing a button is ultimately innocuous.

Reply:
I considered that. A focusable element must _not_ have `aria-hidden="true"`, but the converse is not true, i.e. just because an element isn't focusable doesn't mean it should be hidden from accessibility. From what I was reading, unless the element is purely decorative or would be redundant if announced by a screen reader, we shouldn't hide it. But I'm no expert, so I'm curious what others think, too.

Reply:
At the very least I would think whatever decision we arrived at for these buttons would apply to things like the inc/dec buttons on the number field (which do _not_ provide text content).

Reply:
Sir, [I refute you](https://github.com/ni/nimble/blob/9aec468e4b2c8feabebf39a57632faaa28448f1e/packages/nimble-components/src/number-field/index.ts#L79).

Reply:
Off the top without searching around I can imagine the low-visibility scenario wanting the labels read (moving the mouse cursor around on the screen with a screen reader). Things that are aria-hidden should not be tab focusable but I don't think the inverse is true, i.e. I don't believe something that is not tab focusable needs to be aria-hidden.

Reply:
My guesses might be wrong, the spin button patterns shows the next and previous buttons that are not tab focusable as aria-hidden: https://www.w3.org/WAI/ARIA/apg/patterns/spinbutton/examples/datepicker-spinbuttons/

Reply:
Actually nope, I misread, the buttons in the patterns are not aria-hidden. They have an aria-label and are not tab focusable (tabindex="-1"). What was aria-hidden was helper text on the page showing what the previous and next values are (ie. current month is october and the spinner hides the preview text of september / november)

Reply:
Went ahead and removed the `aria-hidden` settings, and added the labels.